### ISSUE TITLE
Qustion about installation

### ISSUE THREAD

**Issue by LTsommer** [eleurent/highway-env#279]
After install highway_env according to the online documentation, I try to run the simplest code gym.make("highway-v0"), then in the terminal shows connnot from gym.wrappsers import RecordVideo. Has someone else met the same problem? I have checked that gym.wrappers has no module name RecordVideo

**Comment by eleurent** [comment]
Hi! What is your gym version? I think you need at least the 0.21 version (I updated the project so that it works with the latest changes in gym v0.22 and 0.23, which broke compatibility). 

But thanks for the feedback, I will update the package requirements accordingly.

**Comment by LTsommer** [comment]
> Hi! What is your gym version? I think you need at least the 0.21 version (I updated the project so that it works with the latest changes in gym v0.22 and 0.23, which broke compatibility).
> 
> But thanks for the feedback, I will update the package requirements accordingly.

Grateful for your reply. I create a python 3.6 conda environment in order to use GAIL in stable_baselines pkg which use tensorflow 1.15, so I have to degrade gym from 0.21 version to 0.15 version. This is a part of my master thesis, could you pls offer me an old version of highway-env?

**Comment by eleurent** [comment]
Great! It should have worked with gym 0.21 though, so that's a bit weird. I will have a look.
(otherwise, you could also have downgraded to highway-env==1.4 which works with older gym versions <0.21)

**Comment by jsrimr** [comment]
upgrading gym would work

gym==0.17.3
<img width="687" alt="image" src="https://user-images.githubusercontent.com/16870927/159648921-3f37cf15-60b8-4ef7-a1d6-4ee6e82f4202.png">

gym==0.23.1
<img width="749" alt="image" src="https://user-images.githubusercontent.com/16870927/159649015-be48c76f-fbd8-458a-8f92-56a1230d3814.png">

**Comment by jsrimr** [comment]
But tricky thing is that  stable-baselines3 1.4.0 requires gym<0.20,>=0.17.
(stable-baselines3 1.4.0 is the default version via pip install)

**Comment by eleurent** [comment]
Yes exactly, I recommend using their development version (which supports gym 0.21) until their next version is released, as I do in the colab notebooks ([example](https://github.com/eleurent/highway-env/blob/master/scripts/sb3_highway_dqn.ipynb)).

Maybe I should also mention it in the scripts readme
